code/game/objects/items/devices/chameleonproj.dm

```diff
@@ -36,7 +36,7 @@
 	if(!check_sprite(target))
 		return
 	if(!active_dummy)
-		if(isitem(target) && !istype(target, /obj/item/disk/nuclear))
+		if(!isturf(target) && !istype(target, /obj/structure/falsewall) && !iscarbon(target))
```

Comment:
....so you can make yours a mouse or an ash drake?

yeahhhhhhhhhh, no.

Reply:
I agree, I think this steps on morphs toes a bit much if it lets you scan mobs

Reply:
@ShizCalev 
> lettuce not forget the DEFAULT option is a cigbut which is like what 2 pixels?

The only reason you'd scan them is for a joke anyways, you'd literally keep the default option if you were actually using it seriously.

@KorPhaeron that's like saying we shouldn't have chameleon clothes because it interferes with ling's gimmick of changing into anyone on the fly.

Also these guys are defenseless until you take them out of chameleon, morphs are just as powerful in their shiftchange. This probably buffs morphs because people will try to touch chameleon items to reveal players which makes for a mighty tasty snack!

Reply:
Oh and morphs don't have a godawful slowdown.

Reply:
While I agree I want to point out that morphs only have slowdown when undisguised, and do significantly less damage when disguised

Reply:
Aye, cham projector users cannot attack AND have a slowdown when disguised so they get the ~~best~~ worst of both worlds